lib/wootric/test.js

```diff
@@ -0,0 +1,155 @@
+        analytics.assert(window.wootricSettings);
+        analytics.assert(window.wootricSettings instanceof Object)
+      })
+
+      it('should call #load', function(){
+        analytics.called(wootric.load);
+      });
+    });
+  });
+
+  describe('loading', function(){
+    it('should load', function(done){
+      analytics.load(wootric, done);
+    });
+  });
+
+  describe('after loading', function(){
+    beforeEach(function(done){
+      analytics.once('ready', done);
+      analytics.initialize();
+      analytics.page();
+    });
+
+    it('should have created the global wootric object', function(){
+      analytics.assert(window.wootric instanceof Function);
+    })
+
+    it('should have settings with account token', function(){
+      analytics.assert(window.wootricSettings.account_token == 'NPS-01fe3cbc')
+    })
+
+    describe('#identify', function(){
+
+      it('should set email on identify', function(){
+        analytics.identify({
+          email: 'shawn@shawnmorgan.com'
+        });
+        analytics.assert(window.wootricSettings.email, 'shawn@shawnmorgan.com');
+      });
+
+      it('should set created_at on identify using ISO MM/DD/YYYY format', function(){
+        analytics.identify({
+          createdAt: '01/01/2015'
+        });
+        analytics.assert(window.wootricSettings.created_at, '1420099200000');
+      });
+
+      it('should set created_at on identify using ISO YYYY-MM-DD format', function(){
+        analytics.identify({
+          createdAt: '2015-01-01'
+        });
+        analytics.assert(window.wootricSettings.created_at, '1420099200000');
+      });
+
+      it('should set created_at on identify using ISO YYYYMMDD format', function(){
+        analytics.identify({
+          createdAt: '20150101'
+        });
+        analytics.assert(window.wootricSettings.created_at, '1420099200000');
+      });
+
+      it('should set created_at on traits using Unix Timestamp format', function(){
+        analytics.identify({
+          createdAt: '1420099200000'
+        });
+        analytics.assert(window.wootricSettings.created_at, '1420099200000');
+      });
+
+      it('should set properties based on other traits', function(){
+        analytics.identify({
+            email: 'shawn@shawnmorgan.com',
+            createdAt: '01/01/2015',
+            properties: {
+              property1: 'foo',
+              property2: 'bar'
+            }
+        });
+        analytics.assert(window.wootricSettings.properties.property1 === "foo");
+        analytics.assert(window.wootricSettings.properties.property2 === "bar");
+        analytics.assert(!window.wootricSettings.properties.email);
+        analytics.assert(!window.wootricSettings.properties.createdAt);
+      })
+    });
+
+    describe('#page', function(){
+      beforeEach(function(){
+        analytics.initialize();
+      });
+
+      it('should track the current page', function(){
```

Comment:
This test doesn't ensure that `lastPageTracked` points to the correct page, or that it is ever set to `null`. It also doesn't test to be sure that we don't load the pixel in the case that `lastPageTracked` is set. Can you add tests for these cases?

FWIW, this logic should be built into the Wootric library rather than into the integration, but we can handle that later on down the line.